tests/conftest.py

```diff
@@ -49,6 +50,25 @@ def new_user(client, init_testdb):
     return new_user
 
 
+# TODO: How to use the token in other tests?
```

Comment:
Do you mean reuse the token in order tests? Instead of having it create it over and over again? 

If that's the case perhaps you can change the scope of the fixture? 

It is a problem of end to end tests that it's sometimes hard to decide wether you should reset things completely (and log in over and over) or if you allow yourself to stay logged in (opening yourself up to possible interference between tests) 

Doing a complete teardown is safe but slow 😅